src/conversions/hashbrown.rs

```diff
@@ -69,8 +68,8 @@ where
         let dict = PyDict::new(py);
         for (k, v) in self {
             dict.set_item(
```

Comment:
Can this just be `dict.set_item(k, v)` now? Probably a bunch of similar cleanups are now possible.

Reply:
We need to migrate `PyDictMethods` for these, but after that we can clean these up.